phpBB/phpbb/avatar/helper.php

```diff
@@ -252,7 +252,7 @@ private function get_avatar_html(array $data): string
 	{
 		if ($data['lazy'])
 		{
-			$data['src'] = $this->get_no_avatar_source() . ' data-src="' . $data['src'];
+			$data['src'] = $this->get_no_avatar_source() . '" data-src="' . $data['src'];
```

Comment:
```php
		if ($data['lazy'])
		{
			$src = ' src="' . $this->get_no_avatar_source() . '" data-src="' . $data['src'] . '"';
		} else {
  			$src = ' src="' . $data['src'] . '"';
		}
		
		$alt = ' alt="' . $data['title'] . '"';
```

Reply:
This minor fix just needs the missing single quote. I don't think it needs changing beyond that.